.circleci/continue_config.yml

```
@@ -910,6 +910,28 @@ jobs:
           path: /tmp/artifacts
       - <<: *STEP_GET_TEST_AGENT_RESULTS
 
+  test_early_php_81:
+    working_directory: ~/datadog
+    machine:
+      image: ubuntu-2004:2023.04.2
+    steps:
+      - restore_cache:
+          keys:
+            - source-v1-{{ .Branch }}-{{ .Revision }}
+      - <<: *STEP_CHECKOUT
+      - <<: *STEP_ATTACH_WORKSPACE
+      - run:
+          command: |
+            docker run --rm -v $(pwd):/root -i ubuntu:jammy bash -s \<<COMMANDS
+            set -e
+            apt-get update -y
+            DEBIAN_FRONTEND=noninteractive apt-get install -y php8.1 php8.1-dom php-pear
+            cd /root
+            php ./build/packages/datadog-setup.php --php-bin all --file $(ls build/packages/dd-library-php-*-x86_64-linux-gnu.tar.gz)
+            sed -i 's/datadog.trace.request_init_hook/\;datadog.trace.request_init_hook/' /etc/php/8.1/cli/conf.d/98-ddtrace.ini
+            pecl run-tests --showdiff --ini=" -d datadog.trace.cli_enabled=1 -d ddtrace.request_init_hook=" \$(find tests/ext -type d)
```

```suggestion
            pecl run-tests --showdiff --ini=" -d datadog.trace.cli_enabled=1 -d zend.assertions=1" \$(find tests/ext -type d)
```
assertions disabled by default, ugh.